src/website/app/demos/Menu/examples/menu-item/menuItem.js

```diff
@@ -21,14 +21,16 @@ import DemoLayout from '../../components/DemoLayout';
 
 export default {
   id: 'basic',
-  title: 'Basic Menu Item',
+  title: 'Example',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
   description: `
-MenuItems can be used to trigger an action or navigate to a new location.
+MenuItems are used to trigger an action or navigate to a new location.
 
-Note: MenuItems normally occupy the full available width of their container.
-The MenuItems here are width-constrained for illustration purposes.`,
+<Callout title="Note">
```

Comment:
Mind adding this callout to Menu, too? We would have, but `Callout` wasn't in scope before we merged.

Reply:
ah, good idea. added.